app/static/src/app/help/code.md

```diff
@@ -0,0 +1,25 @@
+Write code in this editor and press “Compile” and the odin model will be created, which will be available for use in the other tabs.
+
+### Key bits to remember
+```
+deriv(X) <- ...
+```
+
+Specifies that `X` is a variable that will change over time, but that we can only describe `X` in terms of its rates of 
+change. Every `deriv()` call must be paired with an `initial()` call that describes the initial conditions
+```
+initial(X) <- ...
+```
+The code here looks like R but is not R.
+
+To create other quantities, use `<-` to bind a name (e.g., `a`) to a mathematical expression (e.g., `X + 1`) such as:
+```
```

Comment:
With the current styling, these code blocks don't jump out very clearly. But also happy to defer styling into another ticket